### PR TITLE
Does eccodes really need perl?

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@pelson eccodes has one test, which uses a perl script to compare two result files. If we are sure that this test will actually run OK, than we could just skip this particular test.

**Comment by pelson** [comment]
Thanks @kmuehlbauer. If it needs it, then we should keep it as a dependency. I'll put forward a PR which fixes the pinning.

Thanks,

**Comment by pelson** [comment]
Let's see how that does.

File: ci_support/run_docker_build.sh [hunk 1]
```diff
@@ -44,6 +44,9 @@ conda install --yes conda-build
 conda info
 
 # Embarking on 1 case(s).
+    set -x
+    export CONDA_PERL=5.20.3.1
+    set +x
```
Comment (ocefpaf) [inline ci_support/run_docker_build.sh]: Do we need the set +/- x here? Or they are debug leftovers?

Comment (jakirkham) [inline ci_support/run_docker_build.sh]: This comes out of these [lines]( https://github.com/conda-forge/conda-smithy/blob/211505affe36d8b420ce5ece1860ec401e841daf/conda_smithy/templates/run_docker_build_matrix.tmpl#L12-L14 ) in `conda-smithy`. Given it is just setting environment variable maybe these are not needed.

**Comment by pelson** [comment]
Checks are green. I think we either merge or close. I'm happy either way, but don't want to drag out the PR.

Thanks!

**Comment by ocefpaf** [comment]
We just lost this one under the radar. Thanks from bringing it back.

**Comment by pelson** [comment]
> We just lost this one under the radar. Thanks from bringing it back.

I have plenty of those 😄

**Pull request merged**